site/_docs/docker_images.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: docs
+title: Docker Images
+sidebar_title: Docker Images
+permalink: /docs/docker.html
+---
+
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+## Docker Images for Avatica
+
+[Docker](https://en.wikipedia.org/wiki/Docker_(software)) is a popular piece of
+software that enables other software to run "anywhere". In the context of Avatica,
+we can use Docker to enable a run-anywhere Avatica server. These Docker containers
+can be used to easily create a server for the development of custom Avatica clients
+or encapsulating database access for testing software that uses Avatica.
+
+### Base "avatica-server" Docker Image
+
+Starting with the Avatica 1.10.0 release, Avatica is providing a number of Docker
+containers. Each of these images is based on a "parent" "avatica-server" Docker image.
+
+This Docker image has no bindings to a specific database (it has not database-specific
```

Comment:
Might make sense to include a paragraph about using volumes to add database drivers to the base image? Just a thought since that would be a quick way to test without having to build a dockerfile.